infra/us-east-1/prod/rds/bastion/ec2-instance/terragrunt.hcl

```diff
@@ -0,0 +1,60 @@
+locals {
+
+  # Automatically load environment-level variables
+  account_vars     = read_terragrunt_config(find_in_parent_folders("account.hcl"))
+
+  # Extract out common variables for reuse
+  project        = local.account_vars.locals.project
+  app            = "marlowe-runtime"
+
+  bastion_name       = "${local.project}-${local.app}-database-bastion"
+
+  subnet_id              = "subnet-087617570cd189b3e"
```

Comment:
Probably it's a good idea to fetch this from a dependency instead of hardcoding.

Reply:
Yes, it is being fetched from a dependency, see here: https://github.com/input-output-hk/sc-dev-platform/blob/43cf87dd2ee12ac0d1fdb7ef05c49604989cd7ce/infra/us-east-1/prod/rds/bastion/ec2-instance/terragrunt.hcl#L56. 

Forgot to take this local variable out.